functions/passwords.sh

```diff
@@ -32,26 +33,31 @@ change_password() {
     fi
   else
     passwordChange=$1
-    accounts=("Linux account" "openHAB Console" "Samba")
+    accounts=("Linux system" "openHAB Console" "Samba" "Amanda backup")
   fi
 
   for i in "${accounts[@]}"
   do
-    if [[ $i == *"Linux account"* ]]; then
+    if [[ $i == *"Linux system"* ]]; then
       echo -n "$(timestamp) [openHABian] Changing password for linux account \"$username\"... "
       echo "$username:$passwordChange" | chpasswd
       if [ $FAILED -eq 0 ]; then echo "OK"; else echo "FAILED"; fi
     fi
     if [[ $i == *"Samba"* ]]; then
       echo -n "$(timestamp) [openHABian] Changing password for samba (fileshare) account \"$username\"... "
-      (echo "$passwordChange"; echo "$passwordChange") | /usr/bin/smbpasswd -s -a $username
+      (echo "$passwordChange"; echo "$passwordChange") | /usr/bin/smbpasswd -s -a \"$username\"
       if [ $FAILED -eq 0 ]; then echo "OK"; else echo "FAILED"; fi
     fi
     if [[ $i == *"openHAB Console"* ]]; then
       echo -n "$(timestamp) [openHABian] Changing password for openHAB console account \"openhab\"... "
       sed -i "s/openhab = .*,/openhab = $passwordChange,/g" /var/lib/openhab2/etc/users.properties
       cond_redirect systemctl restart openhab2.service
       if [ $FAILED -eq 0 ]; then echo "OK"; else echo "FAILED"; fi
+    fi
+	if [[ $i == *"Amanda backup account"* ]]; then
```

Comment:
@EliasGabrielsson saw this one second too late. Isn't that a typo?

Reply:
Its totally wrong and on me. Made some last minut changes to the menu to shorten it up. Will make another pull request to fix it.